src/controllers/dynakube/dynakube_controller.go

```diff
@@ -124,7 +120,9 @@ func (controller *DynakubeController) Reconcile(ctx context.Context, request rec
 
 	if dkState.Err != nil {
 		if !dkState.ValidTokens {
-			return reconcile.Result{}, fmt.Errorf("paas or api token not valid")
+			instance.Status.SetPhase(dynatracev1beta1.Error)
+			_ = controller.updateCR(ctx, instance)
+			return reconcile.Result{RequeueAfter: status.DefaultUpdateInterval}, nil
```

Comment:
this feels wrong to access the `defaultUpdateInterval` once here. Would defining it in this file and passing it to the status be an option?

Reply:
that is what we had originally, which was bad as it always the same value was passed it

Reply:
Actually, we could just use `dkState.RequeueAfter`